### ISSUE TITLE
Full Text Search Within a Repo

### ISSUE THREAD

**Issue by jag3773** [unfoldingWord-dev/gogs#57]
Like Github, there should be a search box that allows you to search within files in a repository.

**Comment by jag3773** [comment]
@ethantkoenig This is the next big feature that we need implemented.  Can you check into any upstream implementation or get started on it if no one else has?

**Comment by jag3773** [comment]
@ethantkoenig Any update on this?

**Comment by ethantkoenig** [comment]
@jag3773 I have a ready-to-submit implementation, but it's blocked by the upstream v1.2 release (which has unfortunately been taking longer than expected).

**Comment by jag3773** [comment]
OK, that's what I thought but I wanted to make sure that we recorded that here.  Thanks!

**Comment by ethantkoenig** [comment]
https://github.com/go-gitea/gitea/pull/2582 was merged into master, and will be part of the 1.3 release.
